src/main/java/lotto/domain/LotteryTicket.java

```diff
@@ -10,6 +11,18 @@ public LotteryTicket(List<LottoNumbers> lottoNumbersList) {
         this.lottoNumbersList = lottoNumbersList;
     }
 
+    public LotteryTicket(String[] texts) {
+        this(getLottoNumbers(texts));
+    }
+
+    private static List<LottoNumbers> getLottoNumbers(String[] texts) {
+        List<LottoNumbers> lottoNumbersList = new ArrayList<>();
+        for (String textLottoNumbers : texts) {
+            lottoNumbersList.add(LottoNumbers.of(textLottoNumbers));
+        }
+        return lottoNumbersList;
```

Comment:
```suggestion
        return Arrays.stream(texts)
                .map(LottoNumbers::of)
                .collect(Collectors.toList());
```
이미 알고 계실거 같은데, 혼용을 하셔서 혹시 몰라서 남겼습니다 :)

Reply:
통일성 있게 제안해주신 코드로 변경하는게 좋아보입니다~!!